exo-go/src/cmd/deploy.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/terraform_file_helpers"
+	"github.com/spf13/cobra"
+)
+
+// deployCmd represents the deploy command
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Deploys Exosphere application to the cloud",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
```

Comment:
Yes, it calls a shared method that every other command has been using. Looks like that method leverages the cobra `Help` function.